DirectEmployers/MyJobs
# Separate queries to corresponding components

## What's this PR do?
1. Separates print queries to added print.scss file
2. Separates table queries to table.scss file

## Screenshots (if appropriate)

## Where should the reviewer start?
- cleanup starts in _analytics_media_queries.scss 
- review code added to media query in  _table.scss 
- review print.scss 

## How should this be manually tested?
- Analytics > click to print icon and everything should work as before 
- view charts and tables in mobile view, everything should be responsive as before 

## Any background context you want to provide?
Clean up of _analytics_media_queries before implementing analytics mobile navigation mockup.
